deploy/crds/nmstate.io_v1beta1_nmstate_cr.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: nmstate.io/v1beta1
```

Comment:
Once you rebase, would you please fix it in https://github.com/nmstate/kubernetes-nmstate/blob/master/hack/prepare-release.sh#L54 as well?